source/user-manual/wazuh-indexer/index-life-management.rst

```diff
@@ -1,11 +1,131 @@
+
+      # ISM rollover policy configuration (default values)
+      ism.rollover.enabled: false
+      ism.rollover.index_patterns: ["wazuh-alerts*","wazuh-archives*","-wazuh-alerts-4.x-sample*"]
+      ism.rollover.priority: 50
+      ism.rollover.min_index_age: 7d
+      ism.rollover.min_primary_shard_size: 25gb
+      ism.rollover.min_doc_count: 200000000
+      ism.rollover.overwrite: false
+
+Fortunately, if it's the case that the Wazuh dashboard has already started and applied the default policy to the indices, you can still remove it using the ISM user interface.
+
+To customize your policy, you can follow one of these methods:
+
+-  **Recommended** – Using the ISM user interface.
+-  Using the ISM API.
+-  Using the Wazuh dashboard.
+-  Editing the ``wazuh.yml`` file.
+
+By default, only the ``admin`` and ``kibanaserver`` users have permissions to manage the policies. However, the ``manage_ism`` permission group and role grants permissions to the `ISM API <https://opensearch.org/docs/latest/security/access-control/permissions/#index-state-management-permissions>`__.
+
+Whatever the method you choose to customize the policy, make sure to keep the following considerations in mind:
+
+-  The policy name must be ``wazuh-rollover-policy``.
+-  The ``min_doc_count`` value must be lower than 2^31. Furthermore, we recommend a value not higher than 200 million.
+-  The ``priority`` value must be unique among all policies managing the same indices.
+-  For best performance, we recommend a value between 10 GB and 50 GB for ``min_primary_shard_size``.
+
+.. note::
+   
+   We recommend keeping your documents count from exceeding 200 million while allocating between 10 GB and 50 GB per shard. This configuration typically works well for most use cases. In addition, we don't recommend changing the ``index_patterns`` value.
+
+Using the ISM user interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the recommended method.
+
+#. Click on the upper left menu **☰** and select **Index Management** under **Management**.
+#. Choose **State management policies** and click **wazuh_rollover_policy**. 
+#. Select your preferred edition method and make changes as desired.
+
+Using the ISM API
+~~~~~~~~~~~~~~~~~
+
+You can use the `ISM API <https://opensearch.org/docs/latest/im-plugin/ism/api/>`__ directly from the **Dev Tools** section on the UI. You can also use external tools like `Postman <https://www.postman.com/>`_, `cURL <https://curl.se/>`_, or any other tool or library capable of performing HTTP requests.
+
+Using the Wazuh plugin
+~~~~~~~~~~~~~~~~~~~~~~
+
+#. Click on the upper left menu **☰**, go to **Wazuh**, ...
+
+Editing the ``wazuh.yml`` file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. In the Wazuh dashboard node, edit the ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` file and update with your custom values.
+
+#. Set ``ism.rollover.enabled`` to ``true`` to enable the policy verification during the Wazuh dashboard startup.
```

Comment:
What's a policy verification?

Reply:
The app will verify whether the policy exists during the app's start. If it does not exist, the app will add the policy.

I named it "verification" as I couldn't find a better term.